tests/wallet/nft_wallet/test_nft_1_offers.py

```diff
@@ -1272,7 +1272,17 @@ async def get_trade_and_status(trade_manager: Any, trade: Any) -> TradeStatus:
 )
 @pytest.mark.parametrize(
     "forwards_compat,royalty_pts",
-    [(True, (200, 500, 500)), (False, (200, 500, 500)), (False, (0, 0, 0))],
+    [
+        (True, (200, 500, 500)),
+        (False, (200, 500, 500)),
+        (False, (0, 0, 0)),
+        (False, (9999, 8000, 5000)),
+        (False, (20005, 30005, 40001)),
+        (False, (20000, 30000, 50000)),
+        (False, (65000, 65534, 65535)),
+        (False, (10000, 10001, 10005)),
+        (False, (100000, 10001, 10005)),
```

Comment:
With the parameter above, this is the equivalent of 12 new tests, are they all strictly necessary?

Reply:
forgot to clean up, thanks